tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -0,0 +1,365 @@
+    user enters text into element  id:footnoteForm-content  test footnote from the publication owner! (analyst)
+    user clicks button  Save footnote
+    user waits until h2 is visible  Footnotes
+
+Add public prerelease access list
+    [Tags]  HappyPath
+    user clicks link  Pre-release access
+    user waits until h2 is visible  Manage pre-release user access
+    user creates public prerelease access list   Test public access list
+
+Go to "Sign off" page
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until h2 is visible  Sign off
+
+Assert publication owner cannot approve release for immediate publication
+    [Tags]  HappyPath
+    user checks page does not contain  Edit release status 
+
+Navigate back to release as a publication owner 
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}
+
+Assert that a publication owner can make a new release 
+    [Tags]  HappyPath
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+    user opens accordion section  ${PUBLICATION_NAME}
+    user clicks link  Create new release
+    user creates release for publication  ${PUBLICATION_NAME}  Academic Year Q1  2020
+
+Assert publication owner can upload subject file on new release
+    [Tags]  HappyPath
+    user clicks link  Data and files
+    user waits until page contains element  id:dataFileUploadForm-subjectTitle
+    user enters text into element  id:dataFileUploadForm-subjectTitle   ${SUBJECT_NAME}
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}seven_filters.csv
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}seven_filters.meta.csv
+    user clicks button  Upload data files
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   ${SUBJECT_NAME}
+    user opens accordion section   ${SUBJECT_NAME}
+    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+    user checks headed table body row contains  Status           Complete  ${section}  180
+
+Assert publication owner can add meta guidance to ${SUBJECT_NAME} on new release
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document  90
+    user enters text into element  id:metaGuidanceForm-content  Test meta guidance content
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    user enters text into meta guidance data file content editor  ${SUBJECT_NAME}
+    ...  meta guidance content
+    user clicks button  Save guidance
+
+Go to "Sign off" page as publication owner 
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until h2 is visible  Sign off
+    user checks page does not contain  Edit release status 
+
+Assert publication owner cannot approve release for immediate publication on new release
+    [Tags]  HappyPath
+    user checks page does not contain  Edit release status 
```

Comment:
As previous comment, I think they should be able to request higher level review, and from higher level review go back to draft.